legacy/builder/resolve_library.go

```diff
@@ -31,9 +31,9 @@ func ResolveLibrary(ctx *types.Context, header string) *libraries.Library {
 
 	logger := ctx.GetLogger()
 	if ctx.Verbose {
-		logger.Println(constants.LOG_LEVEL_INFO, fmt.Sprintf(tr("Alternatives for %[1]s: %[2]s"), header, candidates))
+		logger.Println(constants.LOG_LEVEL_INFO, tr("Alternatives for %s:", header), candidates)
```

Comment:
```suggestion
		logger.Println(constants.LOG_LEVEL_INFO, tr("Alternatives for %[1]s: %[2]s", header, candidates))
```
If I managed to follow the code correctly, I think this didn't work as intended because `logger.Println()` eventually passes the `format` and `a...` arguments through `fmt.Sprintf()`, so you get something like `Alternatives for SomeHeader.h:%!(EXTRA string=some candidates)`.